src/components/utils/useListNavigation/README.md

```diff
@@ -0,0 +1,65 @@
+# useListNavigation
+
+React hook used to navigate through the items in a list
+
+`ArrowDown`, `ArrowRight` will increase currently active item index
+`ArrowUp`, `ArrowLeft` will reduce currently active item index
+`PageDown` will increase currently active item index by pageSize (works if `pageSize` passed)
+`PageUp` will reduce currently active item index by pageSize (works if `pageSize` passed)
+`Home` will navigate to the start of the list if `processHomeKey` (disable if you want to move the cursor to the start of active input on `Home` key, for example)
+`End` will navigate to the end of the list if `processEndKey` (disable if you want to move the cursor to the end of active input on `End` key, for example)
+`Enter` or `Space` will select currently active item and call selection callback with that item
```

Comment:
Not sure if `Space` should trigger activation. When used in Dropdown `Space` is usually used on Button to open/close the menu

Reply:
[`Enter` and `Space` are usually used](https://guides.kontur.ru/principles/keyboard/#:~:text=%D0%90%D0%BA%D1%82%D0%B8%D0%B2%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%20%D0%BB%D1%8E%D0%B1%D0%BE%D0%B9%20%D1%8D%D0%BB%D0%B5%D0%BC%D0%B5%D0%BD%D1%82%20%D0%BC%D0%BE%D0%B6%D0%BD%D0%BE%20%D1%81%C2%A0%D0%BF%D0%BE%D0%BC%D0%BE%D1%89%D1%8C%D1%8E%20%D0%BA%D0%BB%D0%B0%D0%B2%D0%B8%D1%88%20Enter%20%D0%B8%D0%BB%D0%B8%C2%A0Space%2C%20%D0%B0%C2%A0%D0%B4%D0%B5%D0%B0%D0%BA%D1%82%D0%B8%D0%B2%D0%B8%D1%80%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%20%D1%81%C2%A0%D0%BF%D0%BE%D0%BC%D0%BE%D1%89%D1%8C%D1%8E%20Enter%2C%20Space%20%D0%B8%D0%BB%D0%B8%C2%A0Esc.) on Button to open/close the menu, but here we have an active item
We can close the menu on Escape